src/index.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+import reportWebVitals from './reportWebVitals';
+import { BrowserRouter } from 'react-router-dom';
+
+ReactDOM.render(
+  <React.StrictMode>
+    <BrowserRouter>
```

Comment:
Also would be good if move BrowserRouter one level dipper into app file or routes

Reply:
In line with what Mohsen noted, may I also suggest having a general understanding of how we built file structure; Components, Components CSS files, images and icons container etc.?